lib/schema.js

```diff
@@ -46,6 +46,26 @@ class Metaschema {
     return errors.length === 0 ? null : new MetaschemaError(errors);
   }
 
+  // Serialize an instance of schema
+  //   type <string>
+  //   schema <Schema> | <string> schema or path, that can be resolved by
+  //       config.resolve
+  //   options <Object>
+  // Returns: <Error> | <null>
+  serialize(type, schema, options) {
+    if (!this.config.processors[type]) {
+      throw new TypeError(`No processors defined for type: '${type}'`);
+    }
+    if (typeof schema === 'string') {
+      schema = this.config.resolve(this, type, schema);
+    }
+    const serializer = this.config.processors[type].serializeSchema;
+    if (!serializer) {
```

Comment:
I'd prefer to first check if serializer exists and only then try to resolve the schema by path (`this.config.resolve(this, type, schema);`).